CITATION.cff

```diff
@@ -0,0 +1,54 @@
+cff-version: 1.2.0
+title: Version Control with Git
+message: >-
+  This lesson is a variation on the Software Carpentry
+  Version Control with Git lesson that includes episodes on
+  branches and pull requests.
+type: dataset
+authors:
+  - given-names: Sarah
+    family-names: Stevens
+    email: sarah.stevens@wisc.edu
+    affiliation: University of Wisconsin-Madison
+    orcid: 'https://orcid.org/0000-0002-7040-548X'
+  - given-names: Patrick
+    family-names: Shriwise
+    orcid: 'https://orcid.org/0000-0002-3979-7665'
+  - given-names: Stephen
+    family-names: Appel
+    orcid: 'https://orcid.org/0000-0002-4976-8489'
+  - given-names: Hoese
+    family-names: David
```

Comment:
These names should be swapped (last name is Hoese). Does this mean my orcid id is wrong? I'm away from a computer for a couple days probably, could someone else fix this for me?